pages/instructorGradebook/instructorGradebook.js

```diff
@@ -21,9 +21,11 @@ var csvFilename = function (locals) {
 };
 
 router.get('/', function (req, res, next) {
-  if (!res.locals.authz_data.has_course_instance_permission_view) {
-    return next(error.make(403, 'Access denied (must be a student data viewer)'));
-  }
+  // We don't actually forbid access to this page if the user is not a student
+  // data viewer, because we want to allow users to click the gradebook tab and
+  // see instructions for how to get student data viewer permissions. Otherwise,
+  // users just wouldn't see the tab at all, and this caused a lot of questions
+  // about why staff couldn't see the gradebook tab.
```

Comment:
This is excellent! But we've seen the same problem happen for pages other than the Gradebook. I remember that people have been confused about why they don't see Downloads and the Students page, in particular. Should we take this opportunity to change all pages to be always visible in navigation (but with the message that this page needs more perms)?

Another case of this that has caused lots of support traffic is the Staff page, where non-Owners can't see it and get confused by the docs that say to go to the Staff page. The same solution would solve this issue.

Reply:
Ahhhh, great call. Yes to all of that! I'll just do that as part of this PR too.

Reply:
Would it make sense here to skip the SQL query in this case, since the data is not needed?